generate_licenses.py

```diff
@@ -219,6 +219,15 @@ def generate_licenses() -> List[License]:
             else:
                 # ライセンスがpypiに無い
                 raise Exception(f"No License info provided for {license.name}")
+
+        # soxr
+        if license.name.lower() == "soxr":
+            with urllib.request.urlopen(
+                "https://raw.githubusercontent.com/dofuuz/python-soxr/v0.3.6/LICENSE.txt"
+            ) as res:
+                license.text += "\n\n----\n\n"
+                license.text += res.read().decode()
```

Comment:
LGPL全文の文章は含まなくても良くて、`LICENSE.txt`の中にあるもので十分そうでした！
なのでこうかなと。
```suggestion
        if license.name.lower() == "soxr":
            with urllib.request.urlopen(
                "https://raw.githubusercontent.com/dofuuz/python-soxr/v0.3.6/LICENSE.txt"
            ) as res:
                license.text = res.read().decode()
```